ServiceInnovationLab/pancake-frontend
# Feature/ui_changes

# What has changed and why?
- UI Changes
- Signature page added

# How to test this change

- Submit a regular rebates form
- in Admin get the token and go to `http://localhost:3000/#/TOKEN` (this is the url sent by email or on paper)
- You should now see the signatures page
- Sign each signature panel and hit submit
- check in admin for signatures

- [x] tested in small screen
- [ ] tested in IE
- [ ] tested in screen reader/contrast <-- remove this if no UX change
